classes/ShoppingfeedApi.php

```diff
@@ -295,29 +295,17 @@ public function getTicketsByReference($taskOrders)
         return $tickets;
     }
 
-    public function getUnacknowledgedOrders($iShipped = false)
+    public function getOrdersFromSf($filters)
     {
-        $status = 'waiting_shipment';
-        if ($iShipped === true) {
-            $status = 'shipped';
-        }
         // Criteria used to query order API
         $criteria = [
-            'filters' => [
-                // Only retrieve unacknowleged (non-imported) orders
-                'acknowledgment' => 'unacknowledged',
-                // Available Shoppingfeed status:
-                // created, waiting_store_acceptance, refused, waiting_shipment, shipped,
-                // cancelled, refunded, partially_refunded, partially_shipped
-                'status' => $status,
-            ],
+            'filters' => $filters,
         ];
 
         Hook::exec(
             'ShoppingfeedOrderImportCriteria', // hook_name
             [
                 'criteria' => &$criteria,
-                'iShipped' => &$iShipped,
```

Comment:
Do not change hook arguments in a minor version because developpers using this hook introduce a breaking changes.